Source/EasyNetQ/Consumer/InternalConsumer.cs

```diff
@@ -29,129 +27,6 @@ IConsumerConfiguration configuration
         event Action<IInternalConsumer> Cancelled;
     }
 
-    public class BasicConsumer : IBasicConsumer, IDisposable
```

Comment:
Is it necessary to move this class? This complicates the code review, you can’t see what really changed.

Reply:
It is not necessary. But i really do think such an important class should not be together with another class. But i can move it back if you think that is best ? :)

Reply:
It really complicates review. We can cleanup code later.

Reply:
done :)